v3/position_test.go

```diff
@@ -27,3 +22,34 @@ var _ = Describe("Position", func() {
 	})
 
 })
+
+var _ = DescribeTable("Should return neighbors of Positions with correct coordinates",
```

Comment:
minor: could you pls just move that within the `Describe("Position", ... ` block?